docs/guides/build-it-yourself/achieve-compliance/deployment-walkthrough/manual_steps.md

```diff
@@ -0,0 +1,129 @@
+
+```bash
+gruntwork aws revoke \
+            --account "dev" \
+            --account "stage" \
+            --account "prod" \
+            --account "security" \
+            --account "shared" \
+            --account "logs"
+```
+
+To revoke Gruntwork's access from the "current" account—the one you are authenticated to in the terminal—use the name
+`__current__` (i.e., `--account "__current__"`).
+
+
+**Important**: Never use the `AdministratorAccess` AWS managed policy with any users, groups, or roles. It gives full access to all resources. Instead, use a policy that allows full IAM privileges (e.g. `iam:*`) on all resources. Our Landing Zone solution created a `iam-admin` group in the Security account.
+
+## AA
+The following steps should be performed on each deployed account.
+
+### 1. Enable MFA for the root user
+
+Securing the "root" user, or the first user that is created when you set up an AWS account, is one of the
+first actions you should take in any new account. It is highly recommended that the user of this account be avoided for everyday tasks. Unfortunately, there is no API or automation available for configuring an MFA device for the
+root user. Follow the manual steps outlined in the
+[AWS docs](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_root-user.html#id_root-user_manage_mfa). Configuring a virtual MFA device will achieve recommendation 1.5. You can also refer to the [production-grade AWS account structure guide.](https://gruntwork.io/guides/foundations/how-to-configure-production-grade-aws-account-structure/)
+
+Configure a hardware MFA device, as suggested by recommendation 1.6. We suggest using a
+[Yubikey](https://www.yubico.com/) due to its reputation for strong security characteristics and multitude of form
+factors. Refer to
+[the documentation for more information on using a hardware device with the root user](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_mfa_enable_physical.html#enable-hw-mfa-for-root).
+
+### 2. Answer security questions and complete contact details
+
+When setting up a new account, AWS asks for contact information and security questions. Unfortunately, there is no API or automation available for this functionality. In the AWS console, visit the [Account settings](https://console.aws.amazon.com/billing/home?#/account) page and complete the _Alternate Contacts_ and _Configure Security Challenge Questions_ questions.
+
+
+### 3. Delete default VPCs and rules from default security groups
+
+Use [cloud-nuke](https://github.com/gruntwork-io/cloud-nuke) to remove the rules from the default VPC and the default ingress/egress rules from the default security groups. Note that it isn’t possible to actually delete the default security group, so instead the command deletes the rules, eliminating the risk of something being mistakenly exposed.
+
+For each account (shared, security, dev, prod etc), run the following command:
```

Comment:
NIT: mention you'll need to authenticate to each account in turn and then run the command.

Reply:
Ah good point. Do you think it's better to have this in the beginning, saying all CLI commands should be authenticated, or each section mentions this? Because one tricky thing that was confusing for me, was that all those steps should be performed for all accounts, so I really want to make sure customers understand that.

Reply:
Good question. You might try: 

1. Stating before the command that you need to authenticate to the correct account before running it 
2. Stating after the command that you must now repeat the previous step for each account, authenticating to dev,stage,prod, etc in turn. 

But I'm not certain - perhaps play around with a couple of different approaches and see which one reads most clearly.